mammoth/utils/statistics.py

```diff
@@ -148,13 +172,17 @@ def output(self, step, num_steps, learning_rate, start, metadata=None):
         meta_str = ''
         if num_steps > 0:
             step_fmt = "%s/%5d" % (step_fmt, num_steps)
+        acc = self.accuracy()
+        acc_str = f'{acc:6.2f}' if acc is not None else '--'
```

Comment:
do we have cases where acc/ppl are none? this is weird

Reply:
`ppl` shouldn't be `None`, unless the model blew up.
I think we check for it somewhere else. If not, we should. Stopping a model blowout isn't the responsibility of the stats logger.

`acc` will be `None` when you don't want to waste compute on it.